TrkDiag/src/BkgDiag_module.cc

```diff
@@ -221,15 +228,19 @@ namespace mu2e
   void BkgDiag::analyze(const art::Event& event ) {
     if(!findData(event))
       throw cet::exception("RECO")<<"mu2e::BkgDiag: data missing or incomplete"<< std::endl;
-    // check consistency,bkgqcol is eliminated from the code, we should add a different check TO DO
-    //if(_bkgccol->size() != _bkgqcol->size())
-    //  throw cet::exception("RECO")<<"mu2e::BkgDiag: data inconsistent"<< std::endl;
+    if(_station){
```

Comment:
You can test directly if you make the expected level a config parameter, see above.